src/screens/Auth/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import {
+  Text,
+  View,
+  TouchableOpacity,
+  Image
+} from 'react-native'
+import styles from '../../Auth/CreatePasswordScreen/styles'
+import BlackHeading from '../../../shared_components/BlackHeading/BlackHeading'
+import GreyTextInput from '../../../shared_components/GreyTextInput/GreyTextInput'
+import PinkButton from '../../../shared_components/PinkButton/PinkButton'
+import { NavigationScreenProp, NavigationState } from 'react-navigation'
+
+interface Props {
+  navigation: NavigationScreenProp<NavigationState>;
```

Comment:
remove the ;
semi-colons in JS are optional, but we aren't using them to stay consistent